src/main/rst/06_test_design_considerations.rst

```diff
@@ -93,14 +93,7 @@ The next time the same page is opened it will likely be a different value.
 Ajax Tests
 ~~~~~~~~~~ 
 
-Ajax is a technology which supports dynamically changing user interface
-elements which can dynamically change without the browser having to reload
-the page, such as animation, RSS feeds, and real-time data updates among others.
-There's a countless ways Ajax can be used to update elements on a web page.
-But, the easy way to think of this is that in Ajax-driven applications, data can
-retrieved from the application server and then displayed on the page without 
-reloading the entire page.  Only a portion of the page, or strictly the element
-itself is reloaded.
+Ajax is a technology which supports dynamically changing user interface elements which can dynamically change without the browser having to reload the page, such as animation, RSS feeds, and real-time data updates among others. There are countless ways Ajax can be used to update elements on a web page. But, the easy way to think of this is that in Ajax-driven applications, data can be retrieved from the application server and then displayed on the page without reloading the entire page. Only a portion of the page, or strictly the element itself is reloaded.
```

Comment:
looks like you didn't change this paragraph, can you revert the removal of the newlines?